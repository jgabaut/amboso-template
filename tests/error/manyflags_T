#!/bin/bash

"./anwrap" -qTibprdgB "0.1.0"
