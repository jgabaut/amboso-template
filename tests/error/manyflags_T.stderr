^[[1;33m[MODE]    Running in test mode.^[[0m$
^[[1;33m[DEBUG]    No -K flag, using ( tests ) for target dir. Run with -V to see more.^[[0m$
^[[1;33m[DEBUG]    bone dir (NO -K passed to this call): ( ok )^[[0m$
^[[1;33m           kulpo dir (NO -K passed to this amboso call): ( error )^[[0m$
$
^[[1;31m[PANIC]    Running  "amboso" using test mode in a program that will be called by test mode is not supported.^[[0m$
$
